cmd/agent/app/check.go

```diff
@@ -79,7 +79,7 @@ var checkCmd = &cobra.Command{
 		}
 
 		s := &serializer.Serializer{Forwarder: common.Forwarder}
-		agg := aggregator.InitAggregatorWithFlushInterval(s, hostname, 10000000000)
+		agg := aggregator.InitAggregatorWithFlushInterval(s, hostname, 1000000000)
```

Comment:
could you use the `checkCmdFlushInterval` constant instead? I don't think we use that constant anywhere otherwise